src/controller/account.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+
+const account = express.Router();
+const { User } = require('../models');
+
+account.get('/user', async (req, res) => {
```

Comment:
You verify the token in the investments endpoint, don't we need it here as well?